x-pack/plugins/siem/public/alerts/containers/detection_engine/alerts/types.ts

```diff
@@ -7,17 +7,17 @@
 export interface BasicSignals {
```

Comment:
I think this one should technically be `BasicAlerts`. Buttt we could also replace it with an inline `{ signal: AbortSignal }` in the few places it's used, too. 🤷‍♂️

Reply:
Yeah, I was 🤷‍♂️ on that as well as I couldn't tell if `BasicSignal` was meant to refer to the `AbortSignal` or SignalSignal. I agree going inline might just be for the best here.